tests/utilities.py

```diff
@@ -87,3 +87,25 @@ def get_treasure_map_from_node(self, node, map_id):
         mock_client = TestClient(node.rest_app)
         return mock_client.get("http://localhost/treasure_map/{}".format(map_id.hex()))
 
+    def ursula_from_rest_interface(self, address, port):
+        for ursula in self.ursulas:
+            if ursula.rest_port == port:
+                rest_app = ursula.rest_app
+                break
+        else:
+            raise RuntimeError("Can't find that one - did you spin up the right test ursulas?")
+        mock_client = TestClient(ursula.rest_app)
+        response = mock_client.get("http://localhost/public_keys")
+        return response
+
+    def get_nodes_via_rest(self, address, port):
```

Comment:
Is this defined outside `MockNetworkyStuff`, or the real version of this method still has to be implemented?

Reply:
Yeah; I'll implement it in `NetworkyStuff` (maybe to be renamed `NetworkMiddleware`?) when I refresh the Finnegan's Wake demo (ie, soon).